asm/macros/event.inc

```diff
@@ -1778,3 +1778,21 @@
 	setfieldeffectargument 2, \priority
 	dofieldeffect FLDEFF_SPARKLE
 	.endm
+
+	@ Prepares to start a double wild battle against a species1 at Level level1 holding item1 and a species2 at Level level2 holding item2. 
+	@ Running this command will not affect normal wild battles. You start the prepared battle with dodoublewildbattle.
+	@ BUG: If the player only has one pokemon, the scripted double battle has undefined behaviours...
```

Comment:
I'd remove this comment and maybe mention seeing the pokeemerald Expansion wiki for a tutorial on how to use it.

Reply:
i don't see why we can't do both. mention that you should verify the player has 2+ mons and also plug the wiki

Reply:
I like Expo's idea  :v

Reply:
Expo's idea of both mentioning the bug and linking to a wiki page makes the most sense so far.  For now though, should we pull and I'll make another PR once the wiki page is up or should the wiki page be made first before and I'll make another commit on this PR?  Also, can both @ultima-soul and @ghoulslash comment on the idea of having mentions of both so I know what to do?

Reply:
Oops. I thought the needing 2 Pokemon for a proper battle was mentioned before the BUG line when it wasn't at all. d:

But yeah, I think rather than highlighting it as a BUG, just mentioning that you should have 2 mons for normal behaviour should be fine and then updating later with a wiki link to a tutorial on how to script a wild double battle properly.

Pyre mentioned different behaviour for handling similar to future gens, but this should be made an issue rather than being commented as a BUG so that we can look into possibilities and research it more.

Reply:
Already made an issue #537 for it a bit ago, the comment was mostly there so if someone finds the bug in the meantime, its already documented as being known.